flocx_market/objects/bid.py

```diff
@@ -0,0 +1,58 @@
+import flocx_market.db.sqlalchemy.api as db
+from flocx_market.objects import base
+from flocx_market.objects import fields
+
+
+class Bid(base.MarketPlaceObject):
+
+    fields = {
+        'marketplace_bid_id': fields.StringField(),
+        'creator_bid_id': fields.StringField(),
+        'creator_id': fields.StringField(),
+        'server_quantity': fields.IntegerField(),
+        'start_time': fields.DateTimeField(nullable=True),
+        'end_time': fields.DateTimeField(nullable=True),
+        'duration': fields.IntegerField(),
+        'status': fields.StringField(),
+        'server_config_query': fields.FlexibleDictField(nullable=True),
+        'cost': fields.FloatField()
+    }
+
+    @classmethod
+    def create(cls, data):
+        bid = db.bid_create(data)
+        return bid.to_dict()
+
+    @classmethod
+    def get(cls, bid_id):
+        if bid_id is None:
+            return cls.get_all()
+        else:
+            b = db.bid_get(bid_id)
+            if b is None:
+                return None
+            else:
+                return b.to_dict()
```

Comment:
We don't want to return a dict; take a look at https://github.com/CCI-MOC/esi-leap/blob/master/esi_leap/objects/base.py#L32-L42 to see one way of returning an Offer object.